src/link_manager/LinkManagerStateMachineActiveActive.h

```diff
@@ -139,6 +139,15 @@ class ActiveActiveStateMachine : public LinkManagerStateMachineBase,
      */
     void handleProbeMuxStateNotification(mux_state::MuxState::Label label) override;
 
+    /**
+     * @method handleControlPlaneConnectionFailure
+     * 
+     * @brief handle gRPC connection failure notification
+     * 
+     * @return none
+     */
+    void handleControlPlaneConnectionFailure() override;
```

Comment:
Would it be better to be named `handleProbeMuxFailure()`?

Reply:
Addressed.